src/backend/storeManagers/gog/games.ts

```diff
@@ -464,9 +465,13 @@ export async function launch(
     ? ['--override-exe', gameSettings.targetExe]
     : []
 
-  let commandEnv = isWindows
-    ? process.env
-    : { ...process.env, ...setupEnvVars(gameSettings) }
+  let commandEnv = {
+    ...process.env,
+    ...setupWrapperEnvVars({ appName, appRunner: 'gog' })
+  }
+  if (!isWindows) {
+    commandEnv = { ...commandEnv, ...setupEnvVars(gameSettings) }
+  }
```

Comment:
```suggestion
  let commandEnv = {
    ...process.env,
    ...setupWrapperEnvVars({ appName, appRunner: 'gog' }),
    ...(isWindows ? {} : setupEnvVars(gameSettings))
  }
```
This feels a little cleaner

Reply:
Good call. Updated in 5d1a6fe7520a240be97faff67fac600f14131d3a.